Finish "Resources" Section (#101)
Slimefun/Wiki: pages/Synthetic-Emerald.md

> The Synthetic Emerald is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used in place of a Vanilla Minecraft emerald for any purposes.

Reviewer: ```suggestion
The Synthetic Emerald is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used instead of a Vanilla Minecraft emerald for all its purposes.
```

## Obtaining

> The Synthetic Emerald can be crafted in the [Smeltery](https://github.com/Slimefun/Slimefun4/wiki/Smeltery) using [1 Aluminum Dust](https://github.com/Slimefun/Slimefun4/wiki/Dusts), [1 Aluminium Ingot](https://github.com/Slimefun/Slimefun4/wiki/Ingots), 1 Glass Pane, and 1 [Synthetic Sapphire](https://github.com/Slimefun/Slimefun4/wiki/Synthetic-Sapphire).

Reviewer: ```suggestion
The Synthetic Emerald can be crafted in the [Smeltery](https://github.com/Slimefun/Slimefun4/wiki/Smeltery) using 1 [Aluminum Dust](https://github.com/Slimefun/Slimefun4/wiki/Dusts), 1 [Aluminium Ingot](https://github.com/Slimefun/Slimefun4/wiki/Ingots), 1 Glass Pane, and 1 [Synthetic Sapphire](https://github.com/Slimefun/Slimefun4/wiki/Synthetic-Sapphire).
```
